README.md

```diff
@@ -1 +1,3 @@
-# pg-activity-watchdog
\ No newline at end of file
+# pg-activity-watchdog
+
+Monitors PostgreSQL activity per client and if the connections counts exceeds a given threshold takes a snapshot as a YAML file.
```

Comment:
Just here a small modification: `takes a snapshot as a JSON and writes to standard output`